detail.py

```diff
@@ -0,0 +1,30 @@
+# from django.conf import settings
+from django_mako_plus import view_function, jscontext
+from datetime import datetime, timezone
+from catalog import models as cmod
+# import math

```

Comment:
Because you have commented these imports out we can probably delete them now.

Reply:
True that. I'll get on that.